Source/WebCore/editing/FrameSelection.cpp

```diff
@@ -2949,8 +2949,8 @@ void FrameSelection::updateFromAssociatedLiveRange()
         disassociateLiveRange();
     else {
         // Don't use VisibleSelection's constructor that takes a SimpleRange, because it uses makeDeprecatedLegacyPosition instead of makeContainerOffsetPosition.
-        auto start = makeContainerOffsetPosition(&m_associatedLiveRange->startContainer(), m_associatedLiveRange->startOffset());
-        auto end = makeContainerOffsetPosition(&m_associatedLiveRange->endContainer(), m_associatedLiveRange->endOffset());
+        auto start = makeContainerOffsetPosition(Ref { m_associatedLiveRange->startContainer() }.ptr(), m_associatedLiveRange->startOffset());
```

Comment:
Again this takes ownership so it seems the argument should not be a raw pointer.